demos/storybook/src/welcome/welcome.component.ts

```diff
@@ -8,12 +8,15 @@ import {hideTopBanner, showTopBanner, UtilModule} from "../utils";
 import * as Colors from '@pxblue/colors';
 const bg = require('../../assets/circles-bg.svg');
 const icon = require('../../assets/pxb-icon.svg');
+import {CommonModule} from '@angular/common';
+import { isDarkMode } from '../utils';
 
 @Component({
     selector: 'welcome',
     styleUrls: ['welcome.component.css'],
     template: `
-        <div class="pxb-blue mat-typography root"
+        <div class="mat-typography root"
```

Comment:
Is dark mode required for the welcome story?

Reply:
probably not... I'll remove it